lib/App/Sqitch/Engine/pg.pm

```diff
@@ -157,7 +157,7 @@ sub initialized {
     my $self = shift;
     return $self->dbh->selectcol_arrayref(q{
         SELECT EXISTS(
-            SELECT TRUE FROM pg_catalog.pg_namespace WHERE nspname = ?
+            SELECT TRUE FROM pg_catalog.pg_tables WHERE schemaname = ? AND tablename = changes
```

Comment:
`changes` needs single quotes around it, `'change'`. Or better make it a `?` and pass the string "changes":

    return $self->dbh->selectcol_arrayref(q{
        SELECT EXISTS(
            SELECT TRUE FROM pg_catalog.pg_namespace WHERE nspname = ? AND tablename = ?
        )
    }, undef, $self->registry, 'changes')->[0];